### PR TITLE
A0: Add missing UI test configurations

### PR DESCRIPTION
**Pull request by jalooc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-import-resolver-typescript": "^3.5.2",
     "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jest": "^27.2.1",
```
Comment (jalooc) [inline package.json:69]: Required for eslint to not throw on the global test functions.

File: package.json [hunk 2]
```diff
@@ -83,7 +84,7 @@
     "predeploy": "yarn run build",
     "deploy": "gh-pages -d build",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!@polkadot)/\"",
```
Comment (jalooc) [inline package.json:87]: Required for jest not to fail for the ES modules syntax in @polkadot packages.

File: .eslintrc.json [hunk 3]
```diff
@@ -82,5 +82,12 @@
     "import/resolver": {
       "typescript": {}
     }
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.test.*"],
```
Comment (deuszx) [inline .eslintrc.json:88]: Didn't know we have tests 🙃

**Review by deuszx — APPROVED** [review]

File: .eslintrc.json (hunk 3 above)
Comment (Kocik) [reply .eslintrc.json:88]: The `npm run test` tells me that 'no tests found'. Is that expected?

**Review by Kocik — APPROVED** [review]

File: .eslintrc.json (hunk 3 above)
Comment (jalooc) [reply .eslintrc.json:88]: Yes - I added the first test in the "dependent" PR: https://github.com/Cardinal-Cryptography/polkadot-staking-dashboard/pull/18

**Pull request merged**